browser/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@0x/mesh-browser",
```

Comment:
@fabioberger what do you think of this name? I also considered just calling it `@0x/mesh` but I want to consider the possibility that we might have a separate, slightly modified version designed to work in Node.js in the future.

Reply:
I like it. Let's stick with it.